Makefile

```diff
@@ -26,6 +26,15 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 	$(PIP_COMPILE) -o requirements/ci.txt requirements/ci.in
 	$(PIP_COMPILE) -o requirements/dev.txt requirements/dev.in
 
+	make upgrade_template
+
+REQ_PATH = "python-template/{{cookiecutter.placeholder_repo_name}}/requirements"
+
+upgrade_template: export CUSTOM_COMPILE_COMMAND=make upgrade
```

Comment:
The words `upgrade_template` don't tell me much about what this target is actually supposed to.  I don't understand how that means "upgrade template pip-tools.txt"

Can you add a comment and/or update the name to more clear about the purpose of this make target.

Reply:
Okay, added the comment. I kept the name the same cause I can't think of anything better in terms of conveying meaning and being concise.